crates/router/src/types/storage/mandate.rs

```diff
@@ -54,6 +54,9 @@ impl MandateDbExt for Mandate {
         if let Some(limit) = mandate_list_constraints.limit {
             filter = filter.limit(limit);
         }
+        if let Some(offset) = mandate_list_constraints.offset {
+            filter = filter.offset(offset);
+        }
```

Comment:
Could you make the corresponding change in the `MockDb` implementation of `MandateInterface` as well?

https://github.com/juspay/hyperswitch/blob/7391416e2473eab0474bd01bb155a9ecc96da263/crates/router/src/db/mandate.rs#L233-L275

Reply:
It should be fixed now! :)